Resources/Help/analyses/correlationbayesian.md

```diff
@@ -44,6 +46,23 @@ The Bayesian Correlation Matrix allows estimation of the population correlation,
 #### Prior
 - Stretched beta prior width: Width of the scaled beta distribution on the correlation under the alterative hypothesis; default is 1. The lower this value, the more concentrated the prior density is around 0. Value must be between 0 and 2.
 
+### Plot Individual Pairs
+- Correlation coefficient to plot:
+  - Pearson's rho: Pearson's product moment correlation coefficient. 
+  - Kendall's tau-b: Kendall's tau-b rank-order correlation coefficient to quantify the monotonic association between two variables.
+  
+- Scatterplot: Displays scatterplots for each specified pair of variables.
+  
+- Prior and posterior: Displays the prior and posterior distribution of the correlation under the alternative hypothesis for each specified pair of variables.
+  - Estimation info: Adds the median and the 95% credible interval of the posterior distribution of the effect size.
+  - Testing info: Adds the Bayes factor computed with the user-defined prior; adds a probability wheel depicting the odds of the data under the null vs. alternative hypothesis; adds indicators representing the prior and posterior at the test point (e.g., rho=0), the ratio between these two values equals the Bayes factor.
```

Comment:
Testing info: Adds the Bayes factor computed with the user-defined prior; adds a probability wheel depicting the odds of the data under the null vs. alternative hypothesis (under the assumption that the null and alternative hypotheses were equal probable a priori, i.e., P(H0)=P(H1)=1/2); adds indicators representing the prior and posterior at the test point (e.g., rho=0), the ratio between these two values equals the Bayes factor.